core/templates/dev/head/pages/exploration_editor/editor_navigation_directive.html

```diff
@@ -70,14 +70,24 @@
     </li>
 
     {% if username %}
-      <li ng-if="isLargeScreen" class="dropdown" popover-is-open="postTutorialHelpPopoverIsShown"
+    <div class="oppia-help-dropdown nav navbar-nav oppia-navbar-nav pull-right ng-cloak">
```

Comment:
@kevinlee12 I have tried making changes so, but the alignment and the navbar-nav settings breaks or popover are not functional when adding it to `dropdown`. Could you please confirm locally? Or if I misinterpreted could you quote the code snippet for clarity. 

Interestingly, popover is acting alike an element here. Modifying it anything else, breaks one or other thing. (nav bar, material icon or the popover itself)